components/chart/pan-zoom.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Pan and Zoom
+page_title: Chart - Pan and Zoom
+description: Learn more about the Pan and Zoom feature of the Telerik UI for Blazor Chart component and explore the available examples.
+slug: components/chart/pan-zoom
+tags: telerik, blazor, chart, event, pan, zoom
+published: true
+position: 31
+---
+
+# Pan and Zoom
+
+`Pan` and `Zoom` allows you to navigate through and zoom in/out the Telerik Blazor Chart component. This functionality works with both `category` and `numeric` series.
+
+## Configuring the Pan Settings
+
+To enable the panning, set the boolean `Enabled` parameter in the corresponding `ChartPannable` inner tag of the Chart. Then, set a range for the axis through the `Min` and `Max` parameters in the corresponding `ChartCategoryAxis` inner tag. 
+
+Charts can be panned to all directions. To perform panning, either: 
+
+* Drag the plot area of the Chart, or
+* (if [key for panning is specified](#specify-key-and-lock-selected-axis-for-panning)) Hold the assigned key and drag the plot area.
+
+>caption Pannable Chart.
+
+````CSHTML
+@* This code snippet showcases an example of pannable Chart. *@
+<TelerikChart>
+    <ChartPannable Enabled="true"></ChartPannable>
+
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Column"
+                     Name="Product 1"
+                     Data="@Data"
+                     Field="@nameof(ChartSeriesData.ProductSales)"
+                     CategoryField="@nameof(ChartSeriesData.Year)">
+        </ChartSeries>
+
+        <ChartCategoryAxes>
+            <ChartCategoryAxis Min="1" Max="5"></ChartCategoryAxis>
+        </ChartCategoryAxes>
+    </ChartSeriesItems>
+</TelerikChart>
+
+@code {
+    List<ChartSeriesData> Data { get; set; } = new List<ChartSeriesData>();
+
+    protected override Task OnInitializedAsync()
+    {
+        Data = ChartSeriesData.GenerateData();
+        return base.OnInitializedAsync();
+    }
+
+    public class ChartSeriesData
+    {
+        public int ProductSales { get; set; }
+        public DateTime Year { get; set; }
+
+        public static List<ChartSeriesData> GenerateData()
+        {
+            List<ChartSeriesData> data = new List<ChartSeriesData>();
+
+            for (int i = 1; i <= 10; i++)
+            {
+                var dataItem = new ChartSeriesData
+                    {
+                        ProductSales = i,
+                        Year = new DateTime(2000 + i, 3, i),
+                    };
+
+                data.Add(dataItem);
+            }
+
+            return data;
+        }
+    }
+}
+````
+
+### Specify Key and Lock Selected Axis for Panning
+
+* **Specify Key** - specify the key for performing panning through the `Key` parameter in the `ChartPannable` tag via the `ChartPannableKey` enum, which provides the following options:
```

Comment:
After splitting the section, this first bullet item can become your introductory sentence for the options that you list below. Don't use bold in this case.